pymongoexplain/commands.py

```diff
@@ -63,10 +92,11 @@ def command_name(self):
 class DistinctCommand(BaseCommand):
     def __init__(self, collection: Collection, key, filter, session,
```

Comment:
Why do some of these classes accept a `session` but some don't? I think they should all be consistent.